tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/repositories/HentKoordinatorerForGjennomforingQuery.kt

```diff
@@ -15,14 +13,12 @@ class HentKoordinatorerForGjennomforingQuery(
 ) {
 
 	private val rowMapper = RowMapper { rs, _ ->
-		Pair(
-			rs.getUUID("gjennomforingId"), Person(
-				fornavn = rs.getString("fornavn"),
-				mellomnavn = rs.getString("mellomnavn"),
-				etternavn = rs.getString("etternavn"),
-				telefonnummer = null,
-				diskresjonskode = null
-			)
+		 Person(
+			fornavn = rs.getString("fornavn"),
+			mellomnavn = rs.getString("mellomnavn"),
+			etternavn = rs.getString("etternavn"),
+			telefonnummer = null,
+			diskresjonskode = null
```

Comment:
Jeg lurer på om vi kunne hatt en egen dbo klasse for dette, siden vi kun bruker navn